test/compress/export.js

```diff
@@ -154,3 +154,46 @@ issue_2134_2: {
         Foo.prototype = {};
     }
 }
+
+redirection: {
+    mangle = {
+        toplevel: true,
+    }
+    input: {
+        let foo = 1;
+        export { foo as var };
+        export { foo as var } from "module.js";
```

Comment:
Isn't `var` the same token type (keyword) as `default`?

Or does `default` has a special meaning as an `export` alias?